com.unity.render-pipelines.universal/Runtime/UniversalRendererData.cs

```diff
@@ -57,6 +57,14 @@ public sealed class ShaderResources
             public Shader materialErrorPS;
         }
 
+        public sealed class TextureResources
+        {
+            [SerializeField, Reload("Textures/numberFont.png")]
+            public Texture2D NumberFont;
+        }
+
+        public TextureResources textures = null;
```

Comment:
Ned public API docs or consider making non public.

Reply:
I think I skipped this due to all other resources in this class also being undocumented :) But that's a poor excuse!

Reply:
Upon closer inspection, this was actually unused and I've removed it. The corresponding resources are now in ScriptableRendererData instead, so I've documented them there.